### ISSUE TITLE
[BUG] Strange behaviour when moving vertex with mirror on

### ISSUE THREAD

**Issue by HoneyBuzzBee** [Inochi2D/inochi-creator#181]
### Validations

- [X] I have checked for similar bug reports and could not find any.
- [X] I have tested and confirmed that this is an issue in an official branded build.

### Describe the bug

Okay, I just found this one while testing for the previous bug. It's an odd one and I'm not sure if I can describe it accurately but the video should clarify what I mean. 
So, when you're editing a mesh in 'Mesh Edit' mode with the horizontal mirror on and you move a vertex, the vertex that is mirrored to it obviously moves too. However, when this mirrored vertex moves over another vertex, the vertex that was initially moving stops and the vertex that it moved over gets dragged around instead. 

### Reproduction

1. Select object with mesh
2. Click 'Edit Mesh'
3. Click 'Mirror horizontally'
4. Select a mirrored vertex
5. Drag the mirrored vertex over another vertex

### System Architecture

x86_64

### Operating System

Linux

### Version

24c7cfb04b1cda17079e235fde8b252b86f33256

### Additional Context

[Screencast from 2023-02-13 02-15-04.webm](https://user-images.githubusercontent.com/121172411/218350884-fffcb39a-0bf5-4069-a864-e818b2999eeb.webm)

**Comment by seagetch** [comment]
I can guess why this happened, but I don't know whether we should fix this or not.

**Comment by seagetch** [comment]
I guess this happened because mirrored vertices are calculated in real time.
Sometimes different vertex is selected as "mirrored", and old vertex might be altered by new one.
This behavior might be useful when user dragged a point and moving around to "capture" the mirrored vertex at beginning.
If application calculate the mirrored vertices only at beginning, this won't happen.
I think if we can discard the behavior above, maybe we can fix this.

**Comment by LunaTheFoxgirl** [comment]
Yeah that's what I'm thinking too. If you don't do it before me I'll try to fix it later

**Comment by seagetch** [comment]
I think prohibiting to exchange already "captured" vertex is a good choice.
If no mirrored vertex is captured, mirror vertices are calculated in real time.
Once vertex is captured, algorithm lookup that vertex, and not searching vertex in real time.
